earthpy/tests/test_plot_rgb.py

```diff
@@ -26,14 +26,25 @@ def rgb_image():
     return rgb, ext
 
 
+def test_no_data_val(rgb_image):
+    """An array with a large no data value should not print nicely, but it should work."""
```

Comment:
This docstring doesn't really describe what's being tested (e.g., we aren't testing that the array is not printing nicely).

Can it be clarified and shortened to fit with our line length limits? e.g., 

```suggestion
    """ Arrays containing -9999 make plots when stretched. """
```